src/App.vue

```diff
@@ -599,6 +601,15 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     await updateLdUser(key, email, firstName, lastName, custom)
   }
 
+  /** Perform high level validations before filing. */
+  private async validateApp (): Promise<void> {
+    // Prompt app validations
+    await this.setAppValidate(true)
```

Comment:
I don't think we have to await setters/actions. We never have before and it's never been a problem.

If you find out otherwise, let everyone know!

(Only NR does this and I'm not sure if that was to work around a problem or what.)

Reply:
Sorry, it doesn't need to be. I was trying a few different things with the validation prompts and left this in. 
Will remove.

Reply:
Removed `await` in next commit 👍